k8s/upgrades/0.8.0-0.8.1/cstor_volume_upgrade.sh

```diff
@@ -97,49 +101,44 @@ c_rs=$(kubectl get rs -o name --namespace $ns | grep $c_dep | cut -d '/' -f 2)
 # previous step                                                #  
 ################################################################
 
-sed "s/@sc_name/$sc_name/g" cstor-target-patch.tpl.json | sed -u "s/@sc_resource_version/$sc_res_ver/g" > cstor-target-patch.json
```

Comment:
I had a comment in the above code that gets c_rs as well (for not to use grep) and gave sample command as well

Reply:
try to do it in my way